code/modules/mob/living/carbon/damage_procs.dm

```diff
@@ -87,7 +87,7 @@
 	if(!forced && HAS_TRAIT(src, TRAIT_TOXINLOVER)) //damage becomes healing and healing becomes damage
 		amount = -amount
 		if(amount > 0)
-			blood_volume -= 3*amount // x5 is too much, x3 should be still penalizing enough.
```

Comment:
This is the only one that I'm hesitant on, it's only in here because the point of this is to yank out every undocumented change.

Need comments on if this should be kept.

Reply:
This was a change I had talked about in public about, more as a mercy.